src/store/selectors/index.js

```diff
@@ -0,0 +1,42 @@
+'use babel'
+
+import { createSelector } from 'reselect'
+import ruleDocURIs from 'eslint-rule-documentation'
+import r from '../../f-utils/mini-ramda'
+import { deepProp } from '../../f-utils'
+import * as rules from './rules'
+import diffDict from '../../f-utils/deep-diff-dict'
+
+
+// Grab any arbitrary property from the state tree with
+// simple string representation { deepKey: 'some.deep.prop' }
+// or null if does not exist.
+//
+export const selectDeepProp = createSelector(
```

Comment:
🔥 This function is never actually used, it only exists here and in the specs for it.